lib/suspenders/generators/app_generator.rb

```diff
@@ -253,5 +253,17 @@ def get_builder_class
     def using_active_record?
       !options[:skip_active_record]
     end
+
+    private
+
+    def honeybadger_outro
+      msg = "Run 'bundle exec honeybadger heroku install' with your API key"
+
+      if options[:heroku]
+        msg << " unless you're using the Heroku Honeybadger add-on"
+      end
+
+      "#{msg}."
```

Comment:
I'm not sure I like how I'm building this outro message here. I want it to end in a period like the rest of the outro no matter which version of the message is used. Is there a cleaner way to do this?